src/index.html

```diff
@@ -15,6 +15,10 @@
       href="https://fonts.googleapis.com/icon?family=Material+Icons"
       rel="stylesheet"
     />
+    <script
+      type="text/javascript"
+      src="https://www.gstatic.com/charts/loader.js"
```

Comment:
we have a library in npm for that - https://www.npmjs.com/package/angular-google-charts. 
Did you check it ? does it fit our needs ?

Reply:
Discussed it during the call